issue_template.md

```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:   
+Editor: TBD  
+Reviewer 1: TBD  
+Reviewer 2: TBD  
+Archive: TBD  
+Version accepted: TBD   
 
+---
 
 -   Paste the full DESCRIPTION file inside a code block below:
 
 ```
 
 ```
 
--   URL for the package (the development repository, not a stylized html page):
 
-- Please indicate which category or categories from our [package fit policies](https://github.com/ropensci/onboarding/blob/master/policies.md#package-fit) this package falls under ***and why**(? (e.g., data retrieval, reproducibility. If you are unsure, we suggest you make a pre-submission inquiry.):
+## Scope 
+
+- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under ***and why**(? (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):
 
-[e.g., "data extraction, because the package parses a scientific data file format"]
+	- [ ] data retrieval
+	- [ ] data extraction
+	- [ ] database access
+	- [ ] data munging
+	- [ ] data deposition
+	- [ ] reproducibility
+	- [ ] geospatial data
+	- [ ] text analysis
```

Comment:
```suggestion
	- [ ] text analysis
	
	- Explain how the and why the package falls under these categories (briefly, 1-2 sentences):
```

Reply:
Suggesting this change because its not clear where and how the author should explain scope.